### ISSUE TITLE
P-TET-000019 Shutoff valve thru body - Design Updates

### ISSUE THREAD

**Issue by mechatroniks-git** [helpfulengineering/project-tetra#98]
- [x] O-ring grooves too tight
- [ ] OD fit with SLA tetra valve was ok, but with FDM shutoff valve, it was a bit tight
- [x] Pocket to hold nut needs to be changed so that nut doesn't spin
- [x] Consider modifying pocket to hold nut such that nut kept centered during assembly (drop in to shallow pocket?)
- [x] Emboss part number on model

**Comment by cyunis** [comment]
Videos from 7/20:
[Archive.zip](https://github.com/tetrabiodistributed/project-tetra/files/4963966/Archive.zip)

**Comment by mechatroniks-git** [comment]
![image](https://user-images.githubusercontent.com/57600622/88311956-69de7900-ccc6-11ea-8bd8-8a74cac69db6.png)

**Comment by jgrano** [comment]
Took all above notes into consideration rework of this part. Will reprint 7/23 and verify fit w/o o-rings

**Comment by Judoguy12** [comment]
John, when you save the part you could put in the reason for saving the
issue number so it is traceable

**Comment by jgrano** [comment]
![image](https://user-images.githubusercontent.com/38054469/88345087-71237800-cd02-11ea-99b2-942d15840d5b.png)
@Judoguy12 so add #98 to the final save?

**Comment by jgrano** [comment]
Final save of design and starting to print based on all feedback.

**Comment by Burhan-Q** [comment]
Darryl to print for testing

**Comment by jgrano** [comment]
Body that I printed works great, integrates fine with existing shutoff valve body.  Waiting for verification from Darryl

**Comment by mechatroniks-git** [comment]
John, nice work. Can I go ahead and print the FDM parts on this?  should I pull STLs from Fusion? Point me in the right direction, want to make sure I print the right ones. Thanks.

**Comment by jgrano** [comment]
https://github.com/tetrabiodistributed/project-tetra/blob/master/02_Development/Prototype_release/Alpha/Parts_to_print/P-TET-000019-2.stl

**Comment by Burhan-Q** [comment]
> https://github.com/tetrabiodistributed/project-tetra/blob/master/02_Development/Prototype_release/Alpha/Parts_to_print/P-TET-000019-2.stl

@DHHwangPhD ready for testing, please comment when print has started

**Comment by mechatroniks-git** [comment]
Part fits knob much better, nut capture is good also. O-ring compression is too much. Need to revise. Currently it is a 2mm o-ring compressed 35%. We should try 10% compression. See https://www.marcorubber.com/o-ring-groove-design-considerations.htm

![image](https://user-images.githubusercontent.com/57600622/89336174-d3a73d00-d64d-11ea-80af-44b4d1fe30b5.png)

Also, not sure we can do much about it, but the thin section at the bottom, but as the groove goes deeper, this will be weaker.

**Comment by mechatroniks-git** [comment]
Reduced o-ring groove ID such that o-ring compression is 10%, not 35%. Should make it easier (possible) to assemble.

Bumped part number embossed on part to P-TET-000019-3

![image](https://user-images.githubusercontent.com/57600622/89357834-7f648300-d676-11ea-936e-6442512f8330.png)

Ready to print & test

**Comment by mechatroniks-git** [comment]
design updated to -3, ready for review @Burhan-Q

**Comment by Burhan-Q** [comment]
> Part fits knob much better, nut capture is good also. O-ring compression is too much. Need to revise. Currently it is a 2mm o-ring compressed 35%. We should try 10% compression. See https://www.marcorubber.com/o-ring-groove-design-considerations.htm
> 
> ![image](https://user-images.githubusercontent.com/57600622/89336174-d3a73d00-d64d-11ea-80af-44b4d1fe30b5.png)
> 
> Also, not sure we can do much about it, but the thin section at the bottom, but as the groove goes deeper, this will be weaker.

![image](https://user-images.githubusercontent.com/62214284/89688309-e03ac800-d8cf-11ea-92ca-623704c53c1c.png)
I'm seeing the change as 1.90 mm (5% of 2mm o-ring). If this is still okay, then I think this part looks good.

General note for main body vs thru body nut capture dimensions, request was to reduce main body nut capture to 7.1 mm [here](https://github.com/tetrabiodistributed/project-tetra/issues/100#issuecomment-669500081), but stated above that this size (7.3 mm) was good.

> Part fits knob much better, nut capture is good also.

**Comment by Burhan-Q** [comment]
@jgrano @Judoguy12 
Ready for approval for whoever is available. Please self assign this issue if you're able to Approve

**Comment by mechatroniks-git** [comment]
@Burhan-Q good catch on the oring groove depth. I'm ok to test it with 5% compression on first article

**Comment by Burhan-Q** [comment]
Latest model milestone
[P-TET-000019-3.zip](https://github.com/tetrabiodistributed/project-tetra/files/5086555/P-TET-000019-3.zip)
